### PR TITLE
Old CentOSes don't support Recommends tag

### PR DESCRIPTION
**Pull request by cbosdo**
## What does this PR change?

Fix CentOS 6 and 7 builds

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [X] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by vzhestkov — APPROVED** [review]
Thanks @cbosdo I missed this point.

File: uyuni/common-libs/uyuni-common-libs.spec [hunk 1]
```diff
@@ -67,7 +67,9 @@ BuildRequires:  python-devel
 %else
 BuildRequires:  python2-devel
 %endif
+%if 0%{?rhel} && 0%{?rhel} < 8
```
Comment (mcalmer) [inline uyuni/common-libs/uyuni-common-libs.spec:70]:
```suggestion
%if 0%{?rhel} >= 8 || 0%{?suse_version}
```
We do not have this package in Debian/Ubuntu Client Tools. So we can ignore them for now.

File: uyuni/common-libs/uyuni-common-libs.spec [hunk 2]
```diff
@@ -85,7 +87,9 @@ Requires:       python3-base
 %else
 Requires:       python3-libs
 %endif
+%if 0%{?rhel} && 0%{?rhel} < 8
```
Comment (mcalmer) [inline uyuni/common-libs/uyuni-common-libs.spec:90]:
```suggestion
%if 0%{?rhel} >= 8 || 0%{?suse_version}
```

**Review by mcalmer — CHANGES_REQUESTED** [review]

File: uyuni/common-libs/uyuni-common-libs.spec (hunk 1 above)
Comment (vzhestkov) [reply uyuni/common-libs/uyuni-common-libs.spec:70]: Yes, this condition makes more sense as `zchunk` is required to exctract metadata by `reposync`.

Comment (cbosdo) [reply uyuni/common-libs/uyuni-common-libs.spec]: doh! I inverted it... fixed

**Review by mcalmer — APPROVED** [review]

**Pull request merged**